backend/utils/parser.py

```diff
@@ -3,14 +3,15 @@
 from typing import List
 from pydantic import BaseModel
 from utils.settings import settings_manager
+from thefuzz import fuzz
 
 
 class ParserConfig(BaseModel):
     language: List[str]
     include_4k: bool
     highest_quality: bool
     repack_proper: bool
-    dual_audio: bool   # This sometimes doesnt work depending on if other audio is in the title
+    dual_audio: bool
```

Comment:
Remove these audio settings aswell

Reply:
This is used for something else and has no effect on if an item gets grabbed or not. This one's fine to stay

Reply:
I can however remove the av1 stuff, and just default to add it

Reply:
fixed.